### PR TITLE
engineering/YLP-1906-Fix-training-modal-CSS

### PR DESCRIPTION
**Pull request by Xwdrblue**

### CODE REVIEW

File: packages/yourloops/pages/training/training.tsx [hunk 1]
```diff
@@ -70,10 +70,10 @@ const style = makeStyles((theme: Theme) => {
     buttons: {
       display: 'flex',
       justifyContent: 'space-around',
-      marginTop: theme.spacing(2),
-      [theme.breakpoints.down('sm')]: {
-        justifyContent: 'space-between'
-      }
+      marginTop: theme.spacing(2)
+      // [theme.breakpoints.down('sm')]: {
```
Comment (TimCanu) [inline packages/yourloops/pages/training/training.tsx:74]: no

**Review by TimCanu — CHANGES_REQUESTED** [review]

File: packages/yourloops/pages/training/training.tsx [hunk 2]
```diff
@@ -70,10 +70,7 @@ const style = makeStyles((theme: Theme) => {
     buttons: {
       display: 'flex',
       justifyContent: 'space-around',
-      marginTop: theme.spacing(2),
-      [theme.breakpoints.down('sm')]: {
-        justifyContent: 'space-between'
-      }
+      marginTop: theme.spacing(2)
```
Comment (fatb38) [inline packages/yourloops/pages/training/training.tsx:73]: `display flex` is not usefull here, you can remove it. Also you could add a little bit more margin top

File: packages/yourloops/pages/training/training.tsx [hunk 3]
```diff
@@ -54,7 +54,7 @@ const style = makeStyles((theme: Theme) => {
     },
     card: {
       textAlign: 'center',
-      padding: theme.spacing(4)
+      padding: theme.spacing(1)
```
Comment (fatb38) [inline packages/yourloops/pages/training/training.tsx:57]: could be removed

**Review by fatb38 — CHANGES_REQUESTED** [review]

File: packages/yourloops/pages/training/training.tsx [hunk 4]
```diff
@@ -68,15 +67,12 @@ const style = makeStyles((theme: Theme) => {
       minHeight: '90vh'
     },
     buttons: {
-      display: 'flex',
       justifyContent: 'space-around',
```
Comment (fatb38) [inline packages/yourloops/pages/training/training.tsx:70]: to remove, useless without `display: flex`
```suggestion
```

**Review by fatb38 — CHANGES_REQUESTED** [review]

**Review by fatb38 — APPROVED** [review]

**Pull request merged**